src/main/java/com/opencart/pages/AdminPage.java

```diff
@@ -0,0 +1,36 @@
+package com.opencart.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AdminPage extends BasePage{
+
+    @FindBy(xpath = ".//*[contains(@href , '#collapse7')]")
+    private WebElement systemButton;
+
+    @FindBy(xpath = ".//*[contains(@href , '#collapse7-2')]")
+    private WebElement localizationButton;
+
+    @FindBy(xpath = ".//*[contains(@href , 'http://localhost/opencart/upload/admin/index.php?route=localisation/currency&user_token=AG2XbPvlJfbHIeiYJAFVx0llKsGXueRi\n')]")
+    private WebElement currenciesButton;
```

Comment:
Could you create a separate page and business logic for NavigationMenu functionality?